tf2onnx/graph.py

```diff
@@ -1124,13 +1124,29 @@ def make_graph(self, doc, graph_name=None, external_tensor_storage=None):
         # create output_tensor_values
         output_tensor_values = self.make_onnx_graph_io(self.outputs)
 
+        tensor_value_info = []
+
+        for op in ops:
+            if op.domain in [constants.ONNX_DOMAIN, constants.AI_ONNX_ML_DOMAIN]:
+                continue
+            # We still don't 100% trust the accuracy of all the shapes in graph.py, but for custom ops they are
+            # almost certainly accurate and onnx has no other way of knowing them.
```

Comment:
Maybe one example would help here. Typical case: Reshape operator, `Z = Reshape(X, Y)` where X and Y are both inputs. There is no way to infer the shape of Z.

Reply:
This case wasn't actually what I was referring to.  Having unknown dims added is fine, the bigger issue would be actually incorrect dims.  While manipulating the graph object I have in the past seen shape data become inaccurate (after it has been used).  We now have tests for this and ideally it should never happen but I'm not 100% sure all bugs have been fixed.